### DISCUSSION TITLE
Remake Infinity to also work with Lemmy/kbin.

### DISCUSSION THREAD

**Discussion by ghost** [Docile-Alligator/Infinity-For-Reddit#1527] [Ideas]
Hi there. Sync has recently remade their app to work with Lemmy, so why not implement a similar thing on at least when adding a new account to the app. The user will be then prompted with a drop-down to select an account type, and if it's Lemmy/kbin, then prompt for an address for the instance. Then, it'll open a web browser that the user can sign in. And that's it.

**Comment by tabtomi8** [comment]
[There's already a Lemmy fork of Infinity. ](https://codeberg.org/Bazsalanszky/Infinity-For-Lemmy)

**Reply by ghost** [reply]
OK, thanks.
